lib/bin/node-spellchecker/package.json

```diff
@@ -16,7 +16,8 @@
     "test": "jasmine-focused --captureExceptions --coffee spec/"
   },
   "devDependencies": {
-    "jasmine-focused": "^1.0.7"
+    "jasmine-focused": "^1.0.7",
+    "node-gyp": "^8.2.0"
```

Comment:
fixing it to latest node-gyp for better compatibility with newer nodejs versions